File: P1/NoMVC/src/TestReadLine.java
import java.io.*;

class TestReadLine {
    public static void main(String[] args) {
        BufferedReader in = new EditableBufferedReader(
                new InputStreamReader(System.in));
        String str = null;
        int n = 0;
        try {
            //str = in.readLine();
            n = in.read();
        } catch (IOException e) { e.printStackTrace(); }
        System.out.println("\nline is: " + n);
    }
}

File: P1/NoMVC/src/TestReadLine.kt
import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

internal object TestReadLine {
    @JvmStatic
    fun main(args: Array<String>) {
        val input: BufferedReader = EditableBufferedReader(InputStreamReader(System.`in`))
        //val str: String = ""
        var n = 0
        try {
            //str = in.readLine();
            n = input.read()
        } catch (e: IOException) {
            e.printStackTrace()
        }
        println("\nline is: $n")
    }
}